frontend/src/tabs/zoom.pug

```diff
@@ -1,8 +1,14 @@
 #zoom
   span.large-font Commits Panel
   .toolbar(v-if="filteredUser.commits.length")
-    a(v-if="expandedCommitMessagesCount === 0", v-on:click="toggleAllCommitMessagesBody(true)") show all commit messages
-    a(v-else, v-on:click="toggleAllCommitMessagesBody(false)") hide all commit messages
+    a(
+      v-if="totalCommitMessageBodyCount !== 0 && expandedCommitMessagesCount === 0",
```

Comment:
LGTM! But I think it might be better to use `v-if` followed by `v-if` and `v-else`.